### DISCUSSION TITLE
[V5] Can i add payload on token to be decoded by the frontend?

### DISCUSSION THREAD

**Discussion by rochagbr** [adonisjs/core#2243] [Help]
Can I do that? How to?
Thankyou.

**Comment by thetutlage** [comment]
Assuming you are talking about the auth opaque tokens. Nope, you cannot do that. The frontend can query the data for the currently logged in user separetely by making an HTTP request

**Reply by rochagbr** [reply]
Thankyou for the fast answer.
Have a good way to use Jwt as auth token on adonis V5?
Then i could to attach user data as payload to it. Right?
